gitlens/gitlens-features.md

```diff
@@ -367,7 +371,7 @@ Commit messages can be automatically generated using [OpenAI](https://openai.com
 
 To connect OpenAI, stage some changes you want to generate a commit message for and then run the `GitLens: Generate Commit Message (Experimental)` command from the Command Palette (`command/ctrl + shift + P`). From there you will be guided through the process of accepting to send the diff of your staged changes to OpenAI and the entry of your OpenAI key. The OpenAI API key can be created [here](https://platform.openai.com/account/api-keys). Once completed, the generated commit message will be entered into the commit input box on the Source Control sidebar. You can also enter additional context about your changes in the commit box and those will also be sent to help generate a better message. Additionally, you can customize the `gitlens.experimental.generateCommitMessagePrompt` setting to control the prompt used to structure and format the generated commit message.
 
-To remove your OpenAI API key in GitLens, run the command `GitLens: Reset Stored OpenAI Key` from the Command Palette.
+To remove your OpenAI API key in GitLens, run the command `GitLens: Reset Stored AI Key` from the Command Palette.
```

Comment:
If I recall correctly, the spirit of this wording change was that we were allowing usage of other AI providers/models (like Anthropic's Claude 2 model). So I'm thinking we may want to audit uses of "OpenAI" throughout our docs and use something more generic like "AI Provider".